modules/Bio/EnsEMBL/G2P/Utils/Downloads.pm

```diff
@@ -54,16 +70,37 @@ sub download_data {
 
   my $csv = Text::CSV->new ( { binary => 1, eol => "\r\n" } ) or die "Cannot use CSV: ".Text::CSV->error_diag ();
   open my $fh, ">:encoding(utf8)", "$file" or die "$file: $!";
+
+  # Write header to file
   $csv->print($fh, ['gene symbol', 'gene mim', 'disease name', 'disease mim', 'DDD category', 'allelic requirement', 'mutation consequence', 'phenotypes', 'organ specificity list', 'pmids', 'panel', 'prev symbols', 'hgnc id', 'gene disease pair entry date']);
 
   $csv->eol ("\r\n");
 
+  # preload phenotypes, organs and publications for all GenomicFeatureDiseases in the database 
   my $gfd_attribute_tables = {
     phenotype => {sql => 'SELECT gfdp.genomic_feature_disease_id, p.stable_id FROM genomic_feature_disease_phenotype gfdp, phenotype p WHERE gfdp.phenotype_id = p.phenotype_id;'},
     organ => {sql => 'SELECT gfdo.genomic_feature_disease_id, o.name FROM genomic_feature_disease_organ gfdo, organ o WHERE gfdo.organ_id = o.organ_id'},
     publication => {sql => 'SELECT gfdp.genomic_feature_disease_id, p.pmid FROM genomic_feature_disease_publication gfdp, publication p WHERE gfdp.publication_id = p.publication_id;'},
   };
 
+  # $gfd_attributes = {
```

Comment:
I like the example but maybe would be better to have the example after the $gfd_attributes is populated.